provision/testdata/list_flavors.stdout.txt

```diff
@@ -16,6 +17,7 @@ INFO: openshift-4.9-openstack:	Red Hat OpenShift Container Platform 4.9 on OpenS
 INFO: openshift-4.8-openstack:	Red Hat OpenShift Container Platform 4.8 on OpenStack
 INFO: openshift-4.7-openstack:	Red Hat OpenShift Container Platform 4.7 on OpenStack
 INFO: openshift-4.6-openstack:	Red Hat OpenShift Container Platform 4.6 on OpenStack
+INFO: openshift-4.14-assisted-esx:	Red Hat OpenShift Container Platform 4.14 on ESX for Assisted Installation
```

Comment:
Please update:
Red Hat OpenShift Container Platform 4.14 on ESX for Assisted Installation
to:
Red Hat OpenShift Container Platform 4.14 on ESX Agent-based Assisted Installation [Experimental]